SPECS/rabbitmq-server/rabbitmq-server.spec

```diff
@@ -0,0 +1,106 @@
+%define  debug_package %{nil}
+Summary:        rabbitmq-server
+Name:           rabbitmq-server
+Version:        3.11.11
+Release:        1%{?dist}
+License:        Apache-2.0 and MPL 2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Development/Languages
+URL:            https://rabbitmq.com
+Source0:        https://github.com/rabbitmq/%{name}/releases/download/v%{version}/%{name}-%{version}.tar.xz
+Source1:        https://github.com/rabbitmq/mix_task_archive_deps/releases/download/1.0.0/mix_task_archive_deps-1.0.0.ez
+Source2:        rabbitmq-server-hex-vendor-%{version}.tar.gz
```

Comment:
I think we should add a comment here on how to leverage the shell script to generate these.

Reply:
Comments added